src/components/organisms/ChatSidebar/components/RecipientChat/RecipientChat.tsx

```diff
@@ -9,10 +9,10 @@ import { WithId } from "utils/id";
 import { Chatbox } from "components/molecules/Chatbox";
 import { UserAvatar } from "components/atoms/UserAvatar";
 
-import { useRecipientChat } from "hooks/privateChats";
-import { useChatSidebarControls } from "hooks/chatSidebar";
+import { useChatSidebarControls } from "hooks/chats/chatSidebar";
 
 import "./RecipientChat.scss";
+import { useRecipientChat } from "../../../../../hooks/chats/privateChats/useRecipientChat";
```

Comment:
same as above

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1922/commits/baab55262718e0f6515911a138c1bfc5504adf49